src/main/java/com/sparta/popupstore/domain/popupstore/service/PopupStoreService.java

```diff
@@ -110,13 +96,25 @@ public PopupStoreUpdateResponseDto updatePopupStore(Long popupId, Company compan
         if (!isEditable(popupStore)) {
             throw new CustomApiException(ErrorCode.POPUP_STORE_ALREADY_START);
         }
-
-        if (imageFile != null) saveImageFile(imageFile);
+        this.updateImage(popupStore, requestDto);
         popupStore.update(requestDto);
 
         return new PopupStoreUpdateResponseDto(popupStoreRepository.save(popupStore));
     }
 
+    private void updateImage(PopupStore popupStore, PopupStoreUpdateRequestDto requestDto) {
+        List<PopupStoreImage> popupStoreImageList = popupStore.getPopupStoreImageList();
+        List<PopupStoreImage> requestImageList = requestDto.getImages().stream()
+                                                                            .map(PopupStoreImageRequestDto::toEntity)
+                                                                            .toList();
+        popupStoreImageList.stream()
+                .filter(image ->
+                        !requestImageList.contains(image)
+                )
+                .forEach(image -> s3ImageService.deleteImage(image.getImageUrl()));
```

Comment:
```suggestion
        popupStoreImageList.stream()
                .forEach(image -> s3ImageService.deleteImage(image.getImageUrl()));
```
어차피 제대로 작동하지 않으니까 빼고 하도록 하죠.